src/Microsoft.DotNet.Darc/DarcLib/Remote.cs

```diff
@@ -90,14 +90,14 @@ public Task CreateOrUpdatePullRequestMergeStatusInfoAsync(string pullRequestUrl,
         return _remoteGitClient.CreateOrUpdatePullRequestMergeStatusInfoAsync(pullRequestUrl, evaluations);
     }
 
-    public async Task<PrStatus> GetPullRequestStatusAsync(string pullRequestUrl)
+    public async Task<PrInfo> GetPullRequestStatusAsync(string pullRequestUrl)
     {
         CheckForValidGitClient();
         _logger.LogInformation($"Getting the status of pull request '{pullRequestUrl}'...");
 
-        PrStatus status = await _remoteGitClient.GetPullRequestStatusAsync(pullRequestUrl);
+        var status = await _remoteGitClient.GetPullRequestStatusAsync(pullRequestUrl);
```

Comment:
Should then this be named `info` instead of `status` (seeing the `status.Status` below)?

Reply:
Applies elsewhere too

Reply:
yeah, my bad, fixed